{"uuid":"b3ec2e27-bc08-4e07-acbe-d73a1127663a","historyId":"d341c415d47c00c9c46beeba6e96298c","fullName":"DataProvider.AddCustomer2.addCustomerTest","labels":[{"name":"package","value":"DataProvider.AddCustomer2"},{"name":"testClass","value":"DataProvider.AddCustomer2"},{"name":"testMethod","value":"addCustomerTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Selenium8AMSept2024"},{"name":"subSuite","value":"DataProvider.AddCustomer2"},{"name":"host","value":"Satyam"},{"name":"thread","value":"4336@Satyam.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"addCustomerTest","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2","trace":"java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:58)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:809)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:734)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2\r\n\tat DataProvider.util.ForDataProvider.getMyData(ForDataProvider.java:43)\r\n\tat DataProvider.AddCustomer2.getData(AddCustomer2.java:59)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54)\r\n\t... 24 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1730738410366,"stop":1730738410369}